### DISCUSSION TITLE
URL contains /index.php/

### DISCUSSION THREAD

**Discussion by datenbursche** [JulianPrieber/littlelink-custom#93] [Issues]
I have installed Littlelink-custom in the root folder of my domain. This now causes the page url to contain /index.php/, something like "tld.com/index.php/@username". Of course I don't like this. Apparently this is related to the .htaccess file, however it is not clear to me how to adjust this. Can anyone help me?

**Comment by JulianPrieber** [comment]
**Hi,**

The index.php normally is not visible in the URL. Routing is handled by Laravel, so the URLs should always be `example.com/`, `example.com/panel/index` ...

Assuming you're using Apache, my first approach would be making sure all required Apache and PHP modules are installed. You can find a list of the basic requirements [here](https://littlelink-custom.com/docs/d/installation-requirements/). 

Make sure you have Apache Module `mod_rewrite` installed and enabled.

Otherwise, please refer to the official Laravel documentation [here](https://laravel.com/docs/8.x) 
Hope this helps :thumbsup:.

**Comment by JulianPrieber** [comment]
Migrated to discussions. If you need further assistance, feel free to join our [community Discord](https://discord.littlelink-custom.com) or simply comment here.
